lib/ex_force.ex

```diff
@@ -271,6 +303,32 @@ defmodule ExForce do
     end
   end
 
+  @doc """
+  Use the Composite API to delete multiple records (up to 200) in one call, returning a list of DeleteResult objects.
+  You can choose whether to roll back the entire request when an error occurs.
+
+  The IDs can belong to different object types, including custom objects.
+
+  While the response code from Salesforce is often a 200, there may be failures mixed with
+  success records so use your own filter and retry logic for error_handling.
+
+  If more than 200 records need to be updated at once, try using the Bulk API:
+  see [Delete Multiple Records with Fewer Round-Trips](https://developer.salesforce.com/docs/atlas.en-us.api_rest.meta/api_rest/resources_composite_sobjects_collections_delete.htm)
+  """
+  @spec delete_sobjects(client, records :: list(sobject), all_or_none :: boolean) ::
+          {:ok, any} | {:error, any}
+  def delete_sobjects(client, ids, all_or_none \\ false)
```

Comment:
I was thinking of pattern matching on the empty set for ids and returning an error which is why I pulled the function signature with the default up

I am not sure what the convention here is or if there is a need to however